frontend/src/static/css/style.scss

```diff
@@ -559,3 +559,20 @@ header{
         text-align:center;
     }
 }
+
+.zoom{
+    &__title{
+        margin-top:1rem;
+
+        &--repo{
+            display:inline;
+            font-size:1.5rem;
+            font-weight:400;
+        }
+
+        &--granularity{
+            display:inline;
+            font-size:1rem;
+        }
```

Comment:
We already have a custom style for smaller font isn't it? can we just reuse that?

Reply:
You mean the `mini-font` class in PR #762 ? The PR is currently still ongoing. 
> actually, instead of `contribution-text-font`, we can just give it a representative name e.g. `mini-font` and make it a global standard in our style sheet.

_Originally posted by @eugenepeh in https://github.com/reposense/RepoSense/pull/762_

Currently there is no global custom style for smaller font. But yeah it would be better if we create a global custom style for `large-font`, `medium-font`, `mini-font`.

Reply:
I see, this can be merged after #762, adopting the same solution then

Reply:
Ok noted.